ui/src/transaction.ts

```diff
@@ -136,36 +136,47 @@ export const formatStake = (msg: Message): string => `${formatMessage(msg)} depo
 export const formatPayer = (identity: identity.Identity): object => {
   return {
     name: identity.metadata.displayName || identity.metadata.handle,
-    variant: "user",
+    variant: "USER",
     avatarFallback: identity.avatarFallback,
     imageUrl: identity.metadata.avatarUrl
   };
 };
 
+export enum SubjectVariant {
```

Comment:
Currently working on a better data structure because `variant` doesn't make a whole lot of sense; for now, this ensures it works.